src/plugins/gauge/components/GaugeComponent.vue

```diff
@@ -24,14 +24,19 @@
     ref="gaugeWrapper"
     class="c-gauge__wrapper js-gauge-wrapper"
     :class="gaugeClasses"
-    :aria-label="gaugeTitle"
     :title="gaugeTitle"
+    :aria-label="`${domainObject.name}`"
```

Comment:
I don't think this is what we want to do here

Reply:
@ozyx can you put a to-do comment to the mage gauges great again card and we'll address this soon